packages/mdc-theme/_functions.scss

```diff
@@ -16,6 +16,11 @@
 
 @import "./constants";
 
+// Used by the functions below to shift a color's lightness by approximately
```

Comment:
I dont think you need these new functions in this PR, can you move it to another one?